### PR TITLE
data:AdityaChaudhary3

### PR DESCRIPTION
**Pull request by AdityaChaudhary3**
## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/5778"></a>

### CODE REVIEW

File: data/AdityaChaudhary3.json [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+  
+  "tags": ["Web Developer", "Open Source", "DevOps"],
+   "socials": [
+    {
+      "icon": "FaTwitter",
+      "url": "https://twitter.com/aditya_script"
+    },
+    {
+      "icon": "FaLinkedin",
+      "url": "https://twitter.com/aditya_script"
```
Comment (khushalarora) [inline data/AdityaChaudhary3.json:14]: I think you want to use your **" Linkedin "** url here :+1: 
```Suggestion
      "url": "https://www.linkedin.com/in/aditya-chaudhary3/"
```

**Review by khushalarora — CHANGES_REQUESTED** [review]
Hi @AdityaChaudhary3, thank you for contributing :tada:

File: data/AdityaChaudhary3.json (hunk 1 above)
Comment (AdityaChaudhary3) [reply data/AdityaChaudhary3.json]: yeah, Thanks for pointing out this.

**Comment by AdityaChaudhary3** [comment]
Thanks for pointing out this.

**Review by SaraJaoude — APPROVED** [review]
looks good

**Pull request merged**